packages/i18n/load-i18n.js

```diff
@@ -1,24 +1,73 @@
 import { addLocaleData } from 'react-intl';
 import { getMatchingMomentCode } from './utils';
 
+const getReactIntlChunkImport = lang => {
+  let chunkImport;
+  switch (lang) {
+    case 'en':
+      chunkImport = import(/* webpackChunkName: "react-intl-data-en" */ 'react-intl/locale-data/en');
+      break;
+    case 'de':
+      chunkImport = import(/* webpackChunkName: "react-intl-data-de" */ 'react-intl/locale-data/de');
+      break;
+    case 'es':
+      chunkImport = import(/* webpackChunkName: "react-intl-data-es" */ 'react-intl/locale-data/es');
+      break;
+    default:
+      chunkImport = import(/* webpackChunkName: "react-intl-data-en" */ 'react-intl/locale-data/en');
+  }
+  return chunkImport;
+};
+
+const getMomentChunkImport = momentLocaleCode => {
+  let chunkImport;
+  switch (momentLocaleCode) {
+    case 'en':
+      chunkImport = import(/* webpackChunkName: "i18n-moment-locale-en-gb" */ 'moment/locale/en-gb');
+      break;
+    case 'de':
+      chunkImport = import(/* webpackChunkName: "i18n-moment-locale-de" */ 'moment/locale/de');
+      break;
+    case 'es':
+      chunkImport = import(/* webpackChunkName: "i18n-moment-locale-es" */ 'moment/locale/es');
+      break;
+    default:
+      chunkImport = import(/* webpackChunkName: "i18n-moment-locale-en-gb" */ 'moment/locale/en-gb');
+  }
+  return chunkImport;
+};
+
+const getLocalizedStringsChunkImport = lang => {
+  let chunkImport;
+  switch (lang) {
+    case 'en':
+      chunkImport = import(/* webpackChunkName: "react-intl-localized-strings-en" */ './data/en.json');
+      break;
+    case 'de':
+      chunkImport = import(/* webpackChunkName: "react-intl-localized-strings-de" */ './data/de.json');
+      break;
+    case 'es':
+      chunkImport = import(/* webpackChunkName: "react-intl-localized-strings-es" */ './data/es.json');
+      break;
+    default:
+      chunkImport = import(/* webpackChunkName: "react-intl-localized-strings-en" */ './data/en.json');
```

Comment:
Lazy question: Do we preserve these comments? They are useful for webpack, so there is value in preserving them.

Reply:
Yes - here is the output in my current rollup branch

```js

var getReactIntlChunkImport = function getReactIntlChunkImport(lang) {
  var reactIntlChunkImport;

  switch (lang) {
    case 'en':
      reactIntlChunkImport = import(
      /* webpackChunkName: "react-intl-data-en" */
      'react-intl/locale-data/en');
      break;
   ...etc
```